src/components/m-dropdown/mixins/dropdown.scss

```diff
@@ -0,0 +1,52 @@
+@import "../settings/dropdown";
+
+@mixin dropdown-container() {
+  position: relative;
+}
+
+@mixin dropdown-toggle() {
+  position: relative;
+
+  &:hover,
+  &:active,
+  &:focus {
+    cursor: pointer;
+  }
+}
+
+@mixin dropdown-icon($size: $dropdown__icon-size, $right: $dropdown__toggle-x-padding) {
+  position: absolute;
+  top: 50%;
```

Comment:
could use calc instead of top:50% and margin-top -(size/2). Much cleaner with calc